index.js

```diff
@@ -71,6 +71,11 @@ function ignoreAuth(req, res) {
         useCreateIndex: true
     })
 
+    var p = 'loopback'
```

Comment:
`var` is deprecated, please use either `const` or `let`.

Reply:
fixed.